storage-management/versioning-limits.md

```diff
@@ -8,6 +8,17 @@ Having to manually keep up with file versions in your workspaces can be unnecess
 
 **Storage Versioning Limits help you limit the number of file versions kept in SharePoint document libraries**. File versions are deleted after exceeding the number of versions you set, the time period you set, or both. 
 
+**A service account must be connected to Syskit Point** to enable:
+* **Automatically applying Versioning limits to workspaces via Rules**
+* **Automatic cleanup actions based on set Versioning limits**
+    * This happens when Syskit Point detects that the file count limit is exceeded or files are older than defined in the Versioning limit
```

Comment:
"file count limit is exceeded or files are older" could just be "file versioning settings are different than defined...."